### PR TITLE
Document the scripts in the hack directory

### PR DESCRIPTION
**Pull request by mrutkows**
# Changes

Added a README.md file to the `hack` directory to describe in more detail each script's purpose, usage, side effects and teardown considerations.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: hack/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Hack
+
+This directory includes convenience scripts for tools that assist developers in creating local Kubernetes clusters (using Docker) and then deploying/configuring Tekton components into them.
+
+## Script overview
+
+| Script | Description |
+| :-- | :-- |
+| [tekton_in_kind.sh](#tekton_in_kind.sh) | Stands up a K8s cluster using the [kind](https://kind.sigs.k8s.io/) tool and deploys Tekton `pipeline`, `triggers` and `dashboard` components |
```
Comment (sbwsg) [inline hack/README.md:9]: small nit: missing `.` at end of sentence.

File: hack/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+See [DEVELOPMENT.md](https://github.com/tektoncd/plumbing/blob/main/DEVELOPMENT.md) for complete complete usage examples.
+
+---
+
+## Script details
+
+### tekton_in_kind.sh
+
+This script uses uses the [`kind` tool](https://kind.sigs.k8s.io/) to create a local K8s cluster in Docker and then deploys [Tekton Pipeline](https://github.com/tektoncd/pipeline), [Tekton Triggers](https://github.com/tektoncd/triggers) and [Tekton Dashboard](https://github.com/tektoncd/dashboard) components, into it.
```
Comment (sbwsg) [inline hack/README.md:20]: nit: `script uses uses` -> `script uses`.

File: hack/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+```shell
+kind delete cluster --name tekton
+```
+
+The `kind` tool will also use the cluster name from the `KIND_CLUSTER_NAME` environment variable if set.
+
+```shell
+export KIND_CLUSTER_NAME=tekton
+$ kind delete cluster
```
Comment (sbwsg) [inline hack/README.md:64]: We haven't included the prompt string (`$`) in any of the other shell blocks.  Suggest either removing it here or adding it in the other code blocks.

**Review by sbwsg — APPROVED** [review]
Cheers @mrutkows !

**Comment by mrutkows** [comment]
@sbwsg Thanks for all the catches of typos/grammar.  Flattened commits and ready for final review (hopefully).

File: hack/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+# Hack
+
+This directory includes convenience scripts for tools that assist developers in creating local Kubernetes clusters (using Docker) and then deploying/configuring Tekton components into them.
+
+## Script overview
+
+| Script | Description |
+| :-- | :-- |
+| [tekton_in_kind.sh](#tekton_in_kindsh) | Stands up a K8s cluster using the [kind](https://kind.sigs.k8s.io/) tool and deploys Tekton `pipeline`, `triggers` and `dashboard` components. |
+| [tekton_ci.sh](#tekton_cish) | Sets up a GitHub webhook to a fork of the `tekton/plumbing` repo. using using the `smee` tool. |
```
Comment (sbwsg) [inline hack/README.md:10]: one more instance of `using using` here but otherwise this looks great!

Comment (mrutkows) [reply hack/README.md]: fixed

**Review by afrittoli — COMMENTED** [review]
Nice, thank you @mrutkows !!
/lgtm

**Pull request merged**